snowpack/src/plugins/plugin-esbuild.ts

```diff
@@ -42,7 +42,7 @@ export function esbuildPlugin(config: SnowpackConfig, {input}: {input: string[]}
         jsxFactory,
         jsxFragment,
         sourcefile: filePath,
-        sourcemap: config.buildOptions.sourcemap,
+        sourcemap: config.buildOptions.sourcemap && 'inline',
```

Comment:
suggestion: it might be cleaner if we just change the accepted values of `config.buildOptions.sourcemap` to be what esbuild accepts `'external' | 'inline' | 'both'`

Reply:
I agree in principle although I couldn't get the browser to pick up any source maps when using any option other than `inline` hence I've limited it to just that at the moment.

Reply:
Interesting, in that case we could just limit `config.buildOptions.sourcemap` to `'inline' | undefined` for now